src/e2e/src/pages/blotter.component.ts

```diff
@@ -1,70 +1,98 @@
 import { by, ElementFinder, ProtractorBrowser } from 'protractor'
 import { mapAsync } from '../utils/async.utils'
-import { waitForElementToBePresent } from '../utils/browser.utils'
+import { waitForElementToBePresent, waitForElementToBeVisible } from '../utils/browser.utils'
 
 export interface BlotterHeader {
   id: string
   label: string
 }
 
+type TradeProperties =
```

Comment:
If we're no longer using the constructor, these typings can be removed, as well as on line `24`

Reply:
This is actually used by a utils file, so this specific comment won't be addressed